packages/auth/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "../../common.tsconfig.json",
+  "include": [
+    "src/**/*.ts"
+  ],
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "strict": false,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "isolatedModules": false,
+    "noEmit": true,
+    "resolveJsonModule": true,
+    "moduleResolution": "NodeNext",
+    "module": "NodeNext",
+    "skipLibCheck": true,
+    "target": "es2020"
+  }
+}
+
```

Comment:
OOS but we should really consider moving a bunch of those settings to be defaults in the common file.